com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -630,7 +631,7 @@ private static void setupProject(Project project, boolean resolveLibraries, IPro
             project.resolveLibUrls(progress);
             TimeProfiler.stop();
         }
-        project.mount(resourceScanner);
+        project.mount(new ClassLoaderResourceScanner());
```

Comment:
Both the editor and Bob can use the same ClassLoaderResourceScanner (it's used to load resources from builtins).